chart/values.yaml

```diff
@@ -2,4 +2,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-
+# Set the controller verbosity
+verbosity:
+  controller: 2
+  agent: 2
```

Comment:
This should probably be transposed; at least the agent will have additional flags that chart users should be able to control (#60 exposes several).